{
	"bugzilla": {
		"description": "CVE-2015-1201 privoxy: denial of service (file descriptor consumption) via unspecified vectors",
		"id": "1203237",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1203237"
	},
	"csaw": false,
	"cvss": {
		"cvss_base_score": "2.6",
		"cvss_scoring_vector": "AV:N/AC:H/Au:N/C:N/I:N/A:P",
		"status": "draft"
	},
	"cwe": "CWE-400",
	"details": [
		"Privoxy before 3.0.22 allows remote attackers to cause a denial of service (file descriptor consumption) via unspecified vectors.  NOTE: the provenance of this information is unknown; the details are obtained solely from third party information."
	],
	"name": "CVE-2015-1201",
	"package_state": [
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:5",
			"fix_state": "Will not fix",
			"package_name": "privoxy",
			"product_name": "Red Hat Enterprise Linux 5"
		}
	],
	"public_date": "2015-01-20T00:00:00Z",
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2015-1201\nhttps://nvd.nist.gov/vuln/detail/CVE-2015-1201"
	],
	"threat_severity": "Low",
	"upstream_fix": "privoxy 3.0.22"
}
